torntools/scripts/content/racing/ttRacingUpgrades.js

```diff
@@ -160,3 +167,11 @@ function resetUpgrades() {
 		}
 	}
 }
+
+function winPercentage() {
+	if (doc.find(".tt-win-percentage")) return;
+	doc.findAll(".enlist-info").forEach((stat)=> {
+		let values = stat.find(".enlisted-stat").innerText.split("• ").filter((x) => x.trim().includes("Races")).map((x) => x.split(":")[1].trim())
+		stat.find(".enlisted-stat").insertAdjacentHTML("beforeEnd", `<span class="tt-win-percentage">• Win Percentage: ${(values[0]/values[1]).toFixed(2)*100} </span>`);
```

Comment:
Better to use a `li` element here, as it's part of a list already anyway.

I would add the % explicitly too.

Reply:
Done.